### ISSUE TITLE
No preview until after closing and re-opening Asciidoc file

### ISSUE THREAD

**Issue by znerd** [asciidoctor/asciidoctor-intellij-plugin#198]
AsciiDoc Plugin Version: 0.19.2

IDE Name and Version: IntelliJ IDEA 2017.3 EAP (Ultimate Edition)
Build #IU-173.3727.22, built on November 15, 2017
IntelliJ IDEA EAP User
Expiration date: December 15, 2017
JRE: 1.8.0_152-release-1024-b6 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13.1

Steps to reproduce:
- Step 1. have an IDEA project with a .adoc file
- Step 2. open the .adoc file
- Step 3. click on the far right icon (in the toolbar right above the editor pane) that represents Preview
- Step 4. click on the icon to the left of it, that represents Editor+Preview

Expected results:
- on step 3, only the preview is shown
- on step 4, the editor (on the left) and the preview (on the right) are shown together

Actual results:
- on step 3, nothing happens
- on step 4, nothing happens

Note: I tried switching between JavaFX and Swing preview, but that does not make a difference.

**Comment by znerd** [comment]
I resolved it by closing the file and then re-opening it.

The cause may have been that I:
- initially assigned `.adoc` to the _Text_ file type,
- then while I had the `.adoc` file open in the editor, …
- …I installed the _AsciiDoc Plugin_
- …and checked the _File Types_ association to confirm that the `.adoc` files are now associated with _Asciidoc_ (which was indeed the case).

Up to you if you consider this something that should be fixed or is low priority.

I'm a happy customer again 👍
